deploy/helm/sumologic/templates/logs/collector/otelcol/daemonset.yaml

```
@@ -13,7 +13,8 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: {{ printf "%s%s" (include "sumologic.metadata.name.logs.collector.daemonset" $ctx) $instance | trunc 63 | trimSuffix "-" }}
-{{- if $.Values.otellogs.daemonset.annotations }}
+  namespace: {{ template "sumologic.namespace" . }}
```

This happens in a loop with a context different than the default. We need to explicitly pass the global context like so:
```suggestion
  namespace: {{ template "sumologic.namespace" $ctx }}
```